lib/components/selects/select.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 import Classlist from 'classlist';
+import CustomEvent from 'custom-event';
```

Comment:
Just curious, why not use the [`Event` constructor](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event) instead?

Reply:
I was scared away from using that based on [IE not supporting it at all](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event#Browser_compatibility)

Reply:
Ah OK. I assume that IE is supported here then?

Reply:
11+ was the (undocumented) original plan. It is a topic that we haven't visited in a while though tbh

Reply:
Gotcha, OK. If we do plan/hope to support IE, then this makes sense to me 👍